docs/source/release_notes/release-notes.rst

```diff
@@ -48,6 +48,12 @@ develop
     *    - Type
          - Change
 
+    *    - |fix|
+         - ``SweepBatchConfig`` values are now decayed correctly when there's an error.
+           ``SweepBatchConfig`` should be decreased until sweep succeeds, however on Cassandra the config was actually multiplied by 1.5.  This was caused by us fixing one of the values at 1.
```

Comment:
In 0.65.1, it wasn't being multiplied by 1.5, with the default `candidateBatchSize` being increased to 1024. So, technically this wasnt the behaviour in the last version.

Reply:
If I'm not wrong this behaviour can still happen with any value of the params, just that it might not happen immediately. 

Even if we start with something big, it will be cut down in the first few iterations until the candidate batch size becomes small.

e.g. running decreases with a batch of `1024`, you get something like:

```
0;0.5;512
1;0.25;256
2;0.125;128
3;0.0625;64
4;0.03125;32
5;0.046875;48
6;0.03125;32
7;0.046875;48
8;0.03125;32
9;0.046875;48
10;0.03125;32
```

and with a batch config of `123456789`:

```
0;0.5;61728394
1;0.25;30864197
2;0.125;15432098
3;0.0625;7716049
4;0.03125;3858024
5;0.015625;1929012
6;0.0078125;964506
7;0.00390625;482253
8;0.001953125;241126
9;9.765625E-4;120563
10;4.8828125E-4;60281
11;2.44140625E-4;30140
12;1.220703125E-4;15070
13;9.953550099535501E-5;12288
14;1.220703125E-4;15070
```

There's probably a proof we can write that the original decreases until it oscillates between K and 1.5K for some K (not going to do it in the interest of time 😅).